depthai_sdk/docs/source/encoding_manager.rst

```diff
@@ -0,0 +1,33 @@
+================
+Encoding manager
+================
+
+``Encoding manager`` is a class that is made to help you with creating videos from OAK cameras.
+
+Getting started
+---------------
+
+Same as ``Preview manager``, ``Encoding manager`` works hand in hand with the ``Pipeline manager``, so before you can use the encoder, you will first have to declare and initialize the ``Pipeline manager``.
+
+.. literalinclude:: ./examples/code_fractions/encoder.py
+   :language: python
+   :linenos:
+
+The most important part, when using the ``Encoding manager`` is that the encoder must be initialized before the pipeline.
```

Comment:
Encoder must be created during pipeline creation, as it adds nodes to the pipeline that gets later executed. So it's best to create encoding manager after PipelineManager, but before sending it to the device